### DISCUSSION TITLE
Override color scheme

### DISCUSSION THREAD

**Discussion by OddSquirrel** [pglombardo/PasswordPusher#632] [Q&A] [answered]
Currently, there is a global blue color scheme for buttons (e.g. "Dashboard", "Push it!").

Is there any way to change that to a different color?

**Comment by pglombardo** [comment]
The only way currently is to manually change the CSS file in the application.  This is acceptable to some but the major drawbacks are:

1. Future application changes may break any user style customisations
2. If using containers, the containers have to be modified or rebuilt entirely to include CSS changes

There is an [open PR currently](https://github.com/pglombardo/PasswordPusher/pull/624) which changes around the CSS packaging.  Once that's merged, I'll detail here best way to modify/extend the CSS.

Longer term, I'd like to have a UI based theme selector potentially.

I'll post back soon once #624 is merged.

**Reply by OddSquirrel** [reply]
Ah, I see. I thought you might be able to expose it as `PWP__ACCENT_COLOR='fff'` or something to that effect.

A long-term solution would be nice, I don't mean to take away from your valuable time.

**Comment by pglombardo** [comment]
Hi @OddSquirrel - I put together some [instructions on how to download and apply any theme](https://github.com/pglombardo/PasswordPusher/issues/762#is) to Password Pusher.  It's a short term work around but works well.

![Screenshot 2023-02-04 at 11 07 40](https://user-images.githubusercontent.com/395132/216762828-90ad3ac6-2f11-402c-9a90-8ba2c00e04ba.png)

**Comment by pglombardo** [comment]
Updating to track.  Built in themes with support for custom CSS coming in https://github.com/pglombardo/PasswordPusher/pull/824

**Comment by pglombardo** [comment, marked as answer]
As of v1.26.10 Password Pusher now

* comes with [26 themes](https://github.com/pglombardo/PasswordPusher/blob/master/Themes.md#theme-gallery) bundled in selectable by environment variable
* also supports [custom CSS](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#how-to-add-custom-css)
* supports [entirely custom themes](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#adding-an-entirely-new-theme-from-scratch)

So with this release you can select a pre-existing theme, create a new theme or add custom CSS.
